### PR TITLE
feat: sync develop into ckb2019-jsbi

### PR DESCRIPTION
**Pull request by sighWang**

### CODE REVIEW

File: packages/common-scripts/tests/common-bigint.tes.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import test from "ava";
```
Comment (homura) [inline packages/common-scripts/tests/common-bigint.tes.ts]: missed filename `test`

File: packages/ckb-indexer/src/transaction_collector.ts [hunk 2]
```diff
@@ -118,22 +118,24 @@ export class CKBIndexerTransactionCollector extends BaseIndexerModule.Transactio
...
+            const targetTx = transactionList.find(
+              (tx) => tx.transaction.hash === transactionHash
+            );
+            if (targetTx) {
+              targetTx.inputCell = output;
+            }
+          });
         });
-      });
+    }
```
Comment (homura) [inline packages/ckb-indexer/src/transaction_collector.ts:138]: Is this fix in the develop branch, if not, this fix should be in a separate PR to ensure that all branches have it

File: yarn.lock [hunk 3]
```diff
@@ -3091,6 +3091,13 @@ babel-plugin-polyfill-regenerator@^0.3.0:
   dependencies:
     "@babel/helper-define-polyfill-provider" "^0.3.0"
 
+babel-plugin-transform-bigint@^1.0.12:
+  version "1.0.12"
+  resolved "https://registry.yarnpkg.com/babel-plugin-transform-bigint/-/babel-plugin-transform-bigint-1.0.12.tgz#d00aff7f9b210faf1853283b798a086956a6d01c"
+  integrity sha512-WJ6BM9ERc8RGv/Oj/UYSs3BX6OCsE4Y70qOh+8c8l4SWc+2js3DmHRTGGbdJXMOtj9UnEKgnr1bfSOvw3GcWuQ==
+  dependencies:
+    "@babel/plugin-syntax-bigint" "^7.8.3"
+
```
Comment (homura) [inline yarn.lock]: I think we don't need the dep here

File: packages/ckb-indexer/src/transaction_collector.ts (hunk 2 above)
Comment (sighWang) [reply packages/ckb-indexer/src/transaction_collector.ts:138]: https://github.com/nervosnetwork/lumos/pull/235

File: yarn.lock (hunk 3 above)
Comment (sighWang) [reply yarn.lock]: removed in next commit
